editoast/src/core/simulation.rs

```diff
@@ -507,9 +501,9 @@ mod tests {
 
     fn create_physics_consist() -> PhysicsConsistParameters {
         PhysicsConsistParameters {
-            total_length: Some(100.0),
+            total_length: Some(Length::new::<meter>(100.0)),
```

Comment:
Interesting that we can use `Velocity` as field types, without the need to specify the unit, but the `new` method **needs** to specify the unit. I'd say it might actually be a good design choice since the function is taking a literal (untyped considering the units)...